Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -149,7 +152,7 @@ class Window final
     NonnullOwnPtr<CSS::Screen> m_screen;
     RefPtr<DOM::Event> m_current_event;
 
-    HashMap<i32, NonnullRefPtr<RequestAnimationFrameCallback>> m_request_animation_frame_callbacks;
+    RequestAnimationFrameDriver m_request_animation_driver;
```

Comment:
That's a weird name, why omit the `frame`? Should be `m_request_animation_frame_driver`.

Reply:
That was just a typo (I sometimes forget to type words and just can't see it). 

But even the name `m_request_animation_frame_driver` seems to read oddly to me, so I've changed it to `m_animation_frame_callback_driver` (and renamed it appropriately in other places).